vignettes/custom-steps.Rmd

```diff
@@ -18,7 +18,8 @@ knitr::opts_chunk$set(
 
 Many important steps running on CI systems are [already implemented](tic.html#steps) into *tic*.
 The following vignette shows how new steps can be created and how *tic* defines them.
-Internally, all steps are defined as [R6](https://github.com/wch/R6) classes.
+Internally, all steps are defined using the [R6](https://github.com/wch/R6) class system.
+For more information see on object-oriented programming using R6 see [this chapter](https://adv-r.hadley.nz/r6.html) in [Advanced R](https://adv-r.hadley.nz/).
```

Comment:
We're mentioning "method", "member" etc. in this section. I'm fine with that, but then an understanding of R6 should be a "formal" prerequisite for this chapter, and not a suggestion. Agree that this isn't the right place for an intro to R6.